File: atividade_nicolas/alunos/nicolas_g1/Kairon/emprestimo.js
import prompt from 'prompt-sync'
const input = prompt({sigint:false})

function main(){
    const rendaMensal = Number(input("Informe sua renda mensal: "))
    const valorEmprestimo = Number(input("Informe o valor do empréstimo(min: R$1.212,00): "))
    const prazo = Number(input("Informe o prazo desejado em meses: "))
    //Processameneto
    const prazoDias = prazo * 30
    const valorIOF = (valorEmprestimo * 0.38 / 100) + (prazoDias * 0.0082 / 100)
    const valorTotalPagar = valorEmprestimo + calcularJuros(valorEmprestimo,valorIOF)
    const valorParcela = valorEmprestimo / prazo


    console.log(`Valor do IOF: R$ ${valorIOF.toFixed(2)}`)
    console.log(`Valor dos juros a pagar: R$ ${calcularJuros(valorEmprestimo,valorIOF).toFixed(2)}`)
    console.log(`Valor total a pagar: R$ ${valorTotalPagar.toFixed(2)}`)
    console.log(`Valor da parcela mensal: ${prazo}X de R$ ${valorParcela.toFixed(2)}`)
    if(valorParcela <= (rendaMensal * 0.4)){
        console.log("Empréstimo aprovado!")
    }else{
        console.log("Empréstimo negado!")    
    }


    function calcularJuros(valorEmprestimo, valorIOF){
        let juros
        if(prazo < 7){
            juros = (valorEmprestimo + valorIOF) * 0.5
        }else if(prazo >= 7 && prazo < 12){
            juros = (valorEmprestimo + valorIOF) * 0.75
        }else if(prazo >= 12 && prazo <= 18){
            juros = (valorEmprestimo + valorIOF)
        }else{
            juros = (valorEmprestimo + valorIOF) * 1.3
        }

        return juros
    }


}
main()

File: atividade_nicolas/alunos/nicolas_g1/kleberson/emprestimo.py
renda_mesnal = int(input('Informe o valor da sua renda mensal: '))
valor_emprestimo = int(input('Informe o valor do emprestimo: '))
prazo = int(input('Informe o prazo: '))
selic = 12.75
prazo_em_dias =  prazo * 30
nova_selic = 0 

iof = (0.38 * valor_emprestimo) + (0.00082 * prazo_em_dias) 

taxa_juros = prazo  * nova_selic
juros_simples = valor_emprestimo * taxa_juros
parcela = (juros_simples / prazo) * 0.4

if prazo <= 6:
    nova_selic = selic * 0.5
    print('Valor do IOF: {}'.format(iof))
    print('Valor do juros a pagar: '.format(taxa_juros))
    print('Valor total a pagar: {}'.format(iof))
    print('Valor da parcela mensal: {}x de {} R$'.format(prazo, parcela))
    print('Valor do IOF: {}.format(iof)'.format)

if prazo <= 12:
    nova_selic = selic * 0.75
    print('Valor do IOF: {}'.format(iof))
    print('Valor do juros a pagar: '.format(taxa_juros))
    print('Valor total a pagar: {}'.format(iof))
    print('Valor da parcela mensal: {}x de {} R$'.format(prazo, parcela))
    print('Valor do IOF: {}'.format)

if prazo >=12 and prazo <= 18:
    nova_selic = selic
    print('Valor do IOF: {}'.format(iof))
    print('Valor do juros a pagar: '.format(taxa_juros))
    print('Valor total a pagar: {}'.format(iof))
    print('Valor da parcela mensal: {}x de {} R$'.format(prazo, parcela))
    print('Valor do IOF: {}'.format)

if prazo > 18:
    nova_selic = selic * 1.3
    print('Valor do IOF: {}'.format(iof))
    print('Valor do juros a pagar: '.format(taxa_juros))
    print('Valor total a pagar: {}'.format(iof))
    print('Valor da parcela mensal: {}x de {} R$'.format(prazo, parcela))
    print('Valor do IOF: {}.format(iof)'.format)

if parcela in renda_mesnal:
    print('Empréstimo aprovado')
else:
    print('Empréstimo negado')